_data/languages.yml

```diff
@@ -71,7 +151,28 @@
   territories: [ bd, in ]
 
 -
-  codes: [ bs, bos ]
+  codes: [ bo, bod, tib ]
+  names: [ Tibetan, Lhasa Tibetan, Standard Tibetan ]
+  family: [ sit ]
+  scripts: [ Tibt ]
+  typology:
+    word_order: [ SOV ]
+    morphosyntax: [ agglutinative ]
+  territories: [ cn, np, in ]
+
+-
+  codes: [ br, bre, xbm, obt ]
+  names: [ Breton ]
+  variant_names: [ Modern Breton, Middle Breton, Old Breton ]
```

Comment:
I think we should remove this, because equivalents exist for so many major languages, like English, Persian etc.